Detections/CommonSecurityLog/TimeSeriesAnomaly-MultiVendor_NetworkTraffic.txt

```diff
@@ -0,0 +1,25 @@
+// Name: Time Series Anomaly detection for Total volume in network logs- MultiVendor
```

Comment:
I reviewed before submission, looks good.

Reply:
Meant to indicate that the query is fine, but lets add some context in such as SourceIP, DestIP, Port and maybe Count of them so it is understood what makes up the spike traffic, this will save the customer time in investigating.

Reply:
Added query secctions to populate more context in results to assist investigations.

Reply:
cool